externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpOps.td

```diff
@@ -61,6 +93,48 @@ def Tcp_AddOp : Tcp_BinaryElementwiseOp<"add", [SameOperandsAndResultElementType
   let assemblyFormat = "$in1 `,` $in2 attr-dict `:` type($in1) `,` type($in2) `->` type($out)";
 }
 
+def Tcp_SubOp : Tcp_BinaryElementwiseOp<"sub", [SameOperandsAndResultElementType]> {
+  let summary = "Computes elementwise addition";
+
+  let description = [{
+    Computes the elementwise subtraction of `in2` from `in1`.
+
+    Same constraints as tcp.add applies to tcp.sub as well.
```

Comment:
Are the type constraints captured in BinaryElementwiseOp? If so should this refer to that instead of add